Kudu.Core/SiteExtensions/SiteExtensionManager.cs

```diff
@@ -92,11 +104,21 @@ public class SiteExtensionManager : ISiteExtensionManager
         private const string _installScriptName = "install.cmd";
         private const string _uninstallScriptName = "uninstall.cmd";
 
+        static SiteExtensionManager()
+        {
+            var aggregateCatalog = new AggregateCatalog();
+            var directoryCatalog = new DirectoryCatalog(HttpRuntime.BinDirectory, "NuGet.Client*.dll");
+            aggregateCatalog.Catalogs.Add(directoryCatalog);
+            _container = new CompositionContainer(aggregateCatalog);
+            _providers = _container.GetExports<INuGetResourceProvider, INuGetResourceProviderMetadata>();
+        }
+
```

Comment:
I would avoid static initializer.  Instead, use Initialize if not initialized appoarch.